src/benchmark/ips.cr

```diff
@@ -79,17 +81,25 @@ module Benchmark
           GC.collect
 
           measurements = [] of Time::Span
+          bytes = 0_i64
+          cycles = 0_i64
+
           target = Time.monotonic + @calculation_time
 
           loop do
+            bytes_before_measure = GC.stats.total_bytes
```

Comment:
Just a thought: Maybe this could be extracted as `GC.measure_total_bytes { ... }`?

Reply:
I don't want to introduce public API, plus it'd used in just one place.